src/components/MapCard/mapCard.module.scss

```diff
@@ -0,0 +1,77 @@
+$primary-color: #f16b67;
+ 
+.cardHeightNone {
+    height: 168px;
+}
+
+.cardHeightButton {
+    height: 240px;
+}
+
+.card {
+    background-color: white;
+    border-radius: 11px;
+    border: 2px solid black;
+    width: 448px;
+    padding: 24px 20px;
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    .cardHeader {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        align-items: flex-end;
+
+        .title {
+            font-size: 24px;
+            font-weight: 900;
+        }
+
+        .difficulty {
+            font-size: 14px;
+            font-weight: 800;
+            
+            .easy {
+                color: #05C05B;
+            }
+
+            .medium {
+                color: #3BB2C8;
+            }
+
+            .hard {
+                color: #D7222C;
+            }
+        }
+    }
+
+    button {
+        width: 142px;
+        height: 40px;
+        border-radius: 6px;
+        border: none;
+        background-color: $primary-color;
+        color: white;
+        display: flex;
+        justify-content: space-evenly;
+        align-items: center;
+
+        // draw a square
+        .arrowBox {
+            width: 26.81px;
+            height: 26.81px;
+            left: 110px;
+            top: 143.7px;
+            background-color: #FFFFFF;
```

Comment:
Make a constant in the global file named `$white` (since it might not literally be white later on for aesthetic/contrast reasons